src/interfaces/IReservoirPriceOracle.sol

```diff
@@ -41,6 +41,7 @@ interface IReservoirPriceOracle {
      */
     function getLatest(OracleLatestQuery calldata priceType) external view returns (uint256);
 
+    // REVIEW: Who calls this?
```

Comment:
advanced users of the oracle, who want to check the upper limit of duration when using the oracle, so as not to get a revert

Reply:
Same as above, can we delete what we're not using?

Reply:
rm-ed